packages/story-editor/src/components/form/tags/input.js

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import { themeHelpers } from '@web-stories-wp/design-system';
+import {
+  useEffect,
+  useMemo,
+  useReducer,
+  useRef,
+  useState,
+} from '@web-stories-wp/react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import { v4 as uuidv4 } from 'uuid';
+/**
+ * Internal dependencies
+ */
+import reducer, { ACTIONS } from './reducer';
+import Tag from './tag';
+
+const INPUT_KEY = uuidv4();
+
+const Border = styled.div`
+  ${({ theme, isInputFocused }) => css`
+    color: ${theme.colors.fg.primary};
+    border: 1px solid ${theme.colors.border.defaultNormal};
+    border-radius: ${theme.borders.radius.small};
+    ${isInputFocused && themeHelpers.focusCSS};
+  `}
+  display: flex;
+  flex-wrap: wrap;
+  padding: 3px 6px;
+  margin-bottom: 6px;
+`;
+
+const TextInput = styled.input.attrs({ type: 'text' })`
```

Comment:
💭 should we use the input from the design system here as a base?

Reply:
ah good call. will update

Reply:
ehh double checking it now, I'm wondering if it has more stuff going on with it than we actually need here

Reply:
maybe I'll just export a base input component from there that's just the text input with the styles instead of all the other stuff that comes with it. should help with not having to augment the DS input at all, but still share the relevant code